src/routes/participants/stats/+page.server.ts

```diff
@@ -29,15 +29,15 @@ const isSponsor = (key: string) =>
 	[
 		'codecentric',
 		'compose-us',
-		'hetzner-logo',
-		'inovex-logo',
+		'hetzner-cloud',
+		'inovex',
 		'jambit',
 		'lary-logo-white',
```

Comment:
I hear @wolframkriesing in my head the whole day now... "Copy and waste."
```suggestion
```

Reply:
Fixed